operator/src/main/java/org/keycloak/operator/controllers/KeycloakController.java

```diff
@@ -234,6 +234,9 @@ public void validatePodTemplate(Keycloak keycloakCR, KeycloakStatusAggregator st
                         status.addWarningMessage(
                                 "The image of the keycloak container cannot be modified using podTemplate");
                     }
+                    if (container.getResources() != null) {
+                        status.addWarningMessage("Resources requirements of the Keycloak container cannot be modified using podTemplate");
```

Comment:
@vmuzikar Not sure, what validation should be done here? The resources should not be specified via podTemplate, when used as first citizen. So, is it sufficient?